docs/source/nlp/text_normalization/wfst/wfst_text_processing_deployment.rst

```diff
@@ -1,81 +1,86 @@
 .. _wfst_deployment:
 
-NeMo Text Processing Deployment
-===============================
+Deploy to Production with C++ backend
+=====================================
 
-NeMo provides a tool for deployment of :doc:`NeMo Inverse Text Normalization (ITN) <wfst_inverse_text_normalization>` and :doc:`NeMo Text Normalization (TN) <wfst_text_normalization>` for production :cite:`textprocessing-deployment-zhang2021nemo`.
-It uses `Sparrowhawk <https://github.com/google/sparrowhawk>`_ :cite:`textprocessing-deployment-sparrowhawk` -- an open-source version of Google Kestrel :cite:`textprocessing-deployment-ebden2015kestrel`.
-The scripts for deployment could be found at `NeMo/tools/text_processing_deployment <https://github.com/NVIDIA/NeMo/tree/main/tools/text_processing_deployment>`_.
+NeMi provides tools to deploy :doc:`TN and ITN <wfst_text_normalization>` for production :cite:`textprocessing-deployment-zhang2021nemo`.
+It uses `Sparrowhawk <https://github.com/google/sparrowhawk>`_ :cite:`textprocessing-deployment-sparrowhawk` -- an open-source C++ framework by Google.
 
 Requirements
 ------------
 
-:doc:`nemo_text_processing <intro>` package
+* :doc:`nemo_text_processing <wfst_text_normalization>` package
+* `Docker <https://www.docker.com/>`_
+* `NeMo source code <https://github.com/NVIDIA/NeMo>`_
 
 
-Usage
------
+.. _wfst_deployment_quick_start:
+Quick Start
+-----------
 
-Starts docker container with production backend with plugged in grammars. This is entry point script.
+Examples how to run: 
 
-Arguments:
-^^^^^^^^^
-* ``GRAMMARS`` - ``tn_grammars`` or ``itn_grammars`` to export either TN or ITN grammars from :doc:`WFST ITN <wfst_inverse_text_normalization>` or :doc:`WFST TN <wfst_text_normalization>`.
-* ``LANGUAGE`` - `en` for English
-* ``INPUT_CASE`` - ``cased`` or ``lower_cased`` (lower_cased is supported only in TN grammars).
-* ``MODE`` - choose ``test`` to run test on the grammars inside the container.
+.. code-block:: bash
 
-For example:
+    # export English TN grammars and return prompt inside docker container  
+    cd NeMo/tools/text_processing_deployment
+    bash export_grammars.sh --GRAMMARS=tn_grammars --LANGUAGE=en --INPUT_CASE=cased
```

Comment:
add a call on how to use the prompt or refer to the export_grammars.sh for more details